core/controllers/reader.py

```diff
@@ -1011,3 +1011,37 @@ def post(self, exploration_id):
             self.payload.get('report_text'),
             self.user_id)
         self.render_json(self.values)
+
+
+class QuestionPlayerHandler(base.BaseHandler):
+    """Provides questions with given skill ids."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.open_access
+    def get(self):
+        """Handles GET request."""
+        start_cursor = self.request.get('start_cursor')
+        """Skill ids are given as a comma separated list because this is
```

Comment:
Use line comments (`# ...`) instead of triple-quoted strings. The latter are typically used only for method-level docstrings (see elsewhere in the codebase).

Reply:
👍